### ISSUE TITLE
Wrog schematic

### ISSUE THREAD

**Issue by xpix** [auspex-nl/esp8266-wifi370#1]
Hello,

i'm a little upset cuz ur'e schematic are wrong :) 

* Ok, please check your MOSFETS in schematic, you need a N-Channel Mosfet instead of P-Channel!
* Please change all Resistor Values in schematic (3x1K and 1x??K R4)

I order as Mosfet or Transistor this one now: http://www.amazon.de/dp/B0087YK2LS/ref=pe_386171_38075861_TE_item
 hope it run :+1:

**Comment by auspex-nl** [comment]
I'm sorry for the confusion about the MOSFETS in the schematic, I used IRLZ34N (n-channel logic gate driven mosfet). I guess I picked the wrong part in the schematic there, but the pins are connected properly (for the IRLZ34N).

That last resistor (R4) is a pullup resistor for the ESP8266, I believe I used 10K for that.

Originally, I planned to write a blog post about this little project with some more information, but I never got round to do that. 

One thing I should probably tell you though is that the voltage regulator will get quite hot without a heatsink. In fact, I miscalculated something when designing this thing and I recommend using a switching power supply instead. 
The ESP8266 uses quite a bit more power than I expected it to.

**Comment by xpix** [comment]
Thats correct, if u use a voltage regulator and you want convert 12V to 3.3V with 250mA ... then it's quite hot. Other Solution. You can use a L7808(8V) and after that a L7803(3V), or you use a step-down regulator: http://www.ebay.nl/itm/LM2596S-3A-DC-DC-Step-Down-SKU8050-Schaltregler-Power-Supply-Modul-Wandler-/151807829999?hash=item235873a7ef ... very cheap, no heat and only 3 bucks :)

Hope this helps!

**Comment by xpix** [comment]
Please check this one also:
https://raw.githubusercontent.com/nodemcu/nodemcu-devkit/master/Documents/NODEMCU_DEVKIT_SCH.png

Thats the nodemcu (very cheap for $5 aliexpress), they use a voiltage regulator for up to 30V!

**Comment by auspex-nl** [comment]
I assumed the power consumption of the ESP8266 would be far less than that 250mA when in idle (connected, but not sending or receiving) state. That SPX3819 would still have some problems dissipating the heat of 250mA if you supply that with 12V input (even with the most efficient packaging, according to the datasheet - although I only did a really quick check, and since my earlier miscalculation, you might want to check for youself ;)). 
I am not exactly sure of this, but I think that nodemcu dev board is supplied with 5V via usb. Which would not be a problem for the 7803 either.

I did order some of those switching step-down regulators some time ago (supercheap, hehe, these: http://www.aliexpress.com/item/10-PCS-DC-DC-Step-Down-Converter-Module-LM2596-DC-4-0-40-to-1-3/1975111111.html). I never got round to finish v2.0 of this board, that besides a proper power supply also has a second microcontroller (attiny) dedicated to a stable and glitch-free PWM output.
